invenio_communities/assets/semantic-ui/js/invenio_communities/search.js

```diff
@@ -0,0 +1,94 @@
+import ReactDOM from "react-dom";
+import React from "react";
+import { Item, Card } from "semantic-ui-react";
+import _truncate from "lodash/truncate";
+import { SearchApp } from "../invenio_search_ui/SearchApp";
+import { overrideStore } from "react-overridable";
+
+function ResultsGridItemTemplate({ result, index }) {
+  return (
+    <Card fluid key={index} href={`/communities/${result.metadata.id}`}>
+      <Card.Content>
+        <Card.Header>{result.metadata.title}</Card.Header>
+        <Card.Description>
+          <div
+            dangerouslySetInnerHTML={{ __html: result.metadata.description }}
+          />
+        </Card.Description>
+      </Card.Content>
+    </Card>
+  );
+}
+
+function ResultsItemTemplate({ result, index }) {
+  return (
+    <Item key={index} href={`/communities/${result.metadata.id}`}>
+      <Item.Content>
+        <Item.Header>{result.metadata.title}</Item.Header>
+        <Item.Description>
+          <div
+            dangerouslySetInnerHTML={{ __html: result.metadata.description }}
+          />
+        </Item.Description>
+      </Item.Content>
+    </Item>
+  );
+}
+
+overrideStore.add("ResultsList.item", ResultsItemTemplate);
+overrideStore.add("ResultsGrid.item", ResultsGridItemTemplate);
+
+const searchConfig = {
```

Comment:
should this be injected from Flask config similar to done in Invenio-Search-UI?

Reply:
I agree, and we discussed with @Glignos that the general mechanism provided by default has to be changed in invenio-search-ui, to allow for extra/hidden parameters, etc. 

The plan, for now, is to get the current PR integrated, so that we have something working with the new `invenio-search-ui` features, and then incrementally improve the "unstable" helpers. We have a list of improvements that we'll convert into issues in `invenio-search-ui`.